core/src/main/scala/org/apache/spark/sql/delta/commands/cdc/CDCReader.scala

```diff
@@ -17,18 +17,15 @@
 package org.apache.spark.sql.delta.commands.cdc
 
 import java.sql.Timestamp
-
```

Comment:
we want a blank line between java and scala

Reply:
IDEA made the change so I followed 🙄